scripts/devnet/lotus.dockerfile

```diff
@@ -11,6 +11,8 @@ RUN CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__" \
     CGO_CFLAGS="-D__BLST_PORTABLE__" \
     make 2k
 
+RUN strip lotus*
```

Comment:
Would it make sense to have it as a part of the previous instruction? This way, we minimize the number of layers.

Reply:
Sure, I can change that.

Reply:
Fixed.